systemtests/scripts/functions

```diff
@@ -3,6 +3,20 @@
 # A set of useful functions to be sourced in each test
 #
 
+wait_for_jobs_to_terminate ()
+{
+  status_of_what=$1
+  max_wait_in_seconds=$2
+  count=max_wait_in_seconds
+  while (( --count >= 0 )); do
+    if  "${BAREOS_BCONSOLE_BINARY}" -c "${conf}" <<< "status ${status_of_what}" | grep "No Jobs running.";
```

Comment:
This is not the most recent version.  This currently has the problem that it will timeout the test if the daemon crashed since the bconsole invocation takes ~15sec and you do it 20 times.

On master it was changed to
```diff
modified   systemtests/scripts/functions
@@ -7,8 +7,8 @@ wait_for_jobs_to_terminate ()
 {
   status_of_what=$1
   max_wait_in_seconds=$2
-  count=max_wait_in_seconds
-  while (( --count >= 0 )); do
+  start_time=$SECONDS
+  while (( SECONDS - start_time <= max_wait_in_seconds )); do
     if  "${BAREOS_BCONSOLE_BINARY}" -c "${conf}" <<< "status ${status_of_what}" | grep "No Jobs running.";
     then
       break
```